docs/verifiable-document/dns-proof.md

```diff
@@ -4,8 +4,60 @@ title: Configuring DNS
 sidebar_label: Configuring DNS
```

Comment:
I'm also wondering if we shouldnt emphasize on the dns value ? like in the previous step.

Reply:
Sorry I dont get this one

Reply:
Like in this page => https://openattestation.com/docs/verifiable-document/document-store

we outline the important of the document store value, I'm wondering if we should find a way to outline the importance of the domain used

Reply:
hmm dont think this is super important cause it's easy to get it back. Smart contract addresses are harder to get back